test/datatypes/interval-test.ts

```diff
@@ -33,7 +33,7 @@ describe('Interval', () => {
 });
 
 describe('DateTimeInterval.contains', () => {
-  let d;
+  let d: any;
```

Comment:
(Looking at this happen dozens of times below...) Oh man, that's annoying.  Sorry.  No need to change it now, but we probably could have wrapped these all in a suite and just put the `beforeEach` in that instead.  Or I think you could have just put `let d: any;` in the module scope.

Reply:
eh, no sweat